### PR TITLE
Fix question display bug and add to collab page

### PR DESCRIPTION
**Pull request by tyanhan**
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
image

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings

### CODE REVIEW

File: frontend/src/components/Question.js [hunk 1]
```diff
@@ -50,9 +52,9 @@ const Question = () => {
 	useEffect(() => {
 		getQuestion();
 	}, []);
-
+	console.log(question);
```
Comment (tau-bar) [inline frontend/src/components/Question.js:55]: Remove `console.log`
```suggestion

```

**Comment by tau-bar** [comment]
LGTM, some minor nits:
1. Remove `console.log` statements in the `Question` component.
2. This button 
image
It seems faded, maybe can make it more prominent (it seems like it is a disabled button). Also the box shadow is oval?

**Review by tau-bar — CHANGES_REQUESTED** [review]

**Comment by tyanhan** [comment]
image
Changed the button colour

**Review by tau-bar — APPROVED** [review]
LGTM

**Pull request merged**